Packs/CommonScripts/ReleaseNotes/1_12_12.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### StixCreator
+
+- Fixed an issue in exporting `Report` indicator in STIX format.
```

Comment:
what was the issue? can you make it to the standard?
"ixed an issue where something failed when something was ran..."